core/services/chainlink/application.go

```diff
@@ -290,11 +307,50 @@ func NewApplication(config *orm.Config, ethClient eth.Client, advisoryLocker pos
 		}}
 		headTrackables = append(headTrackables, headTrackable)
 	}
-	app.HeadTracker = services.NewHeadTracker(store, headTrackables)
+	app.HeadTracker = services.NewHeadTracker(headTrackerLogger, store, headTrackables)
+	serviceLoggers[logger.HeadTracker] = app.HeadTracker
 
 	return app, nil
 }
 
+// SetServiceLogger sets the logger for a given service
+func (app *ChainlinkApplication) SetServiceLogger(serviceName string, level string) error {
+	var ll zapcore.Level
```

Comment:
probably just want to accept a zapcore.Level as the argument here and do the marshalling / error checking in the log controller and then we dont marshal it back and forth twice